app/javascript/elm/tests/MainTests.elm

```diff
@@ -138,7 +138,7 @@ locationFilter =
                 { defaultModel | location = location }
                     |> update SubmitLocation
                     |> Tuple.second
-                    |> Expect.equal (Ports.findLocation location)
```

Comment:
Personally I'd consider removing all those tests. Not sure what value they add. Also, when they fail the error message is not clear.